tox.ini

```diff
@@ -94,4 +94,5 @@ commands =
     nosetests --rednose --immediate -sv contrib/runners/local_runner/tests/integration/
     nosetests --rednose --immediate -sv contrib/runners/mistral_v2/tests/integration/
     nosetests --rednose --immediate -sv contrib/runners/orquesta_runner/tests/integration/
+	nosetests --rednose --immediate -sv st2tests/integration/orquesta/
```

Comment:
Extra whitespace, but it's not end of the world :P

Reply:
Done